firejail/virtualbox.profile

```diff
@@ -0,0 +1,22 @@
+# virtualbox profile
```

Comment:
The files in firejail I believe are by our custom design. If not, I'll remove them myself; however, I do believe we did push changes to them.